widget/entry_test.go

```diff
@@ -366,6 +366,26 @@ func TestEntry_Tapped_AfterRow(t *testing.T) {
 	assert.Equal(t, 0, entry.CursorColumn)
 }
 
+func TestEntry_TappedSecondary(t *testing.T) {
```

Comment:
This is really testing the paste rather than the tappedSecondary.
It should either be asserting that menus appear, or it should have a name that reflects the fact it is testing pasteFromClipboard.